base/src/main/java/com/smartdevicelink/proxy/rpc/OnVehicleData.java

```diff
@@ -458,11 +467,29 @@ public Double getInstantFuelConsumption() {
         return SdlDataTypeConverter.objectToDouble(object);
     }
 
+    /**
+     * Sets the externalTemperature.
+     *
+     * @param externalTemperature The external temperature in degrees celsius. This parameter is deprecated starting RPC
+     * Spec 7.1, please see climateData.
+     * @since SmartDeviceLink 2.0.0
+     * @deprecated in SmartDeviceLink 7.1.0
+     */
+    @Deprecated
     public OnVehicleData setExternalTemperature(Double externalTemperature) {
         setParameters(KEY_EXTERNAL_TEMPERATURE, externalTemperature);
         return this;
     }
 
+    /**
+     * Gets the externalTemperature.
+     *
+     * @return Boolean The external temperature in degrees celsius. This parameter is deprecated starting RPC

```

Comment:
@kostyaBoss On line 487 this should be `Float` instead of `Boolean`.

Reply:
@kostyaBoss I checked again with the generator and it says `Float`.  Since you are having issues with the `Float`, you can just make it `Double`.